frontend/src/components/Homepage/Homepage/Homepage.tsx

```diff
@@ -26,6 +26,11 @@ const everyoneItems: readonly NavigationCardItem[] = [
     header: 'Team Event Credits',
     description: 'Track your team event credits.',
     link: '/forms/teamEventCredits'
+  },
```

Comment:
See my other comment about using `adminOnly` here and hiding the card for non-admins while we're still developing this.